src/shared/i18n/I18NFrench.cpp

```diff
@@ -18,441 +18,441 @@
-    { I18N_TNMT_SR_BEST, "best:" },
-    { I18N_TNMT_SR_LAST, "last:" },
-    { I18N_TNMT_SR_NA, "N/A" },
-    { I18N_TNMT_PLAYER_JOINS, "${1} joins the game" },
-    { I18N_TNMT_PLAYER_FELL_OFF, "${1} fell off the stage" },
-    { I18N_TNMT_PLAYER_SUICIDE, "${1} committed suicide" },
-    { I18N_TNMT_COIN_RETURNED, "the coin returned to its base" },
-    { I18N_TNMT_COIN_CATCHED, "${1} catched the coin" },
-    { I18N_TNMT_COIN_DROPPED, "the coin dropped" },
-    { I18N_TNMT_RED_FLAG_RETURNED, "the red flag returned to its base" },
-    { I18N_TNMT_BLUE_FLAG_RETURNED, "the blue flag returned to its base" },
-    { I18N_TNMT_RED_FLAG_CATCHED, "${1} catched the red flag" },
-    { I18N_TNMT_BLUE_FLAG_CATCHED, "${1} catched the blue flag" },
-    { I18N_TNMT_RED_FLAG_SAVED, "the red flag returned by ${1}" },
-    { I18N_TNMT_BLUE_FLAG_SAVED, "the blue flag returned by ${1}" },
-    { I18N_TNMT_RED_FLAG_DROPPED, "the red flag dropped" },
-    { I18N_TNMT_BLUE_FLAG_DROPPED, "the blue flag dropped" },
-    { I18N_TNMT_TEAM_RED_SCORED1, "team red scores" },
-    { I18N_TNMT_TEAM_BLUE_SCORED1, "team blue scores" },
-    { I18N_TNMT_TEAM_RED_SCORED2, "TEAM RED SCORES" },
-    { I18N_TNMT_TEAM_BLUE_SCORED2, "TEAM RED SCORES" },
-    { I18N_TNMT_SR_RECORD, "new record of ${1}: ${2}" },
+    { I18N_TNMT_SB_SR_TITLE, "COURSE" },
+    { I18N_TNMT_SB_LAPS, "TOURS" },
+    { I18N_TNMT_SB_BEST, "MEILLEUR" },
+    { I18N_TNMT_SB_LAST, "DERNIER" },
+    { I18N_TNMT_SR_LEAD, "meneur:" },
+    { I18N_TNMT_SR_LAP, "tour:" },
+    { I18N_TNMT_SR_BEST, "meilleur:" },
+    { I18N_TNMT_SR_LAST, "dernier:" },
+    { I18N_TNMT_SR_NA, "" },
+    { I18N_TNMT_PLAYER_JOINS, "${1} à rejoint la partie" },
+    { I18N_TNMT_PLAYER_FELL_OFF, "${1} est tombé d'un étage" },
+    { I18N_TNMT_PLAYER_SUICIDE, "${1} a choisi le suicide" },
+    { I18N_TNMT_COIN_RETURNED, "la pièce est de nouveau dans sa base" },
+    { I18N_TNMT_COIN_CATCHED, "${1} a attrapé la pièce" },
+    { I18N_TNMT_COIN_DROPPED, "la pièce a été lachée" },
+    { I18N_TNMT_RED_FLAG_RETURNED, "le drapeau rouge est retourné à sa base" },
+    { I18N_TNMT_BLUE_FLAG_RETURNED, "le drapeau bleu est retourné à sa base" },
+    { I18N_TNMT_RED_FLAG_CATCHED, "${1} a attrapé le drapeau rouge" },
+    { I18N_TNMT_BLUE_FLAG_CATCHED, "${1} a attrapé le drapeau bleu" },
+    { I18N_TNMT_RED_FLAG_SAVED, "le drapeau rouge a été sauvé par ${1}" },
+    { I18N_TNMT_BLUE_FLAG_SAVED, "le drapeau bleu a été sauvé par ${1}" },
+    { I18N_TNMT_RED_FLAG_DROPPED, "le drapeau rouge a été laché" },
+    { I18N_TNMT_BLUE_FLAG_DROPPED, "le drapeau bleu a été laché" },
+    { I18N_TNMT_TEAM_RED_SCORED1, "l'équipe rouge a marqué" },
+    { I18N_TNMT_TEAM_BLUE_SCORED1, "l'équipe bleue a marqué" },
+    { I18N_TNMT_TEAM_RED_SCORED2, "L'ÉQUIPE ROUGE MARQUE" },
+    { I18N_TNMT_TEAM_BLUE_SCORED2, "L'ÉQUIPE BLEUE MARQUE" },
+    { I18N_TNMT_SR_RECORD, "nouveau record de ${1}: ${2}" },
     { I18N_TNMT_SR_COMPLETE, "round completed" },
```

Comment:
"fin de la manche"? Or might need to be tested in context for a more accurate translation.